### ISSUE TITLE
Add "insecure" option for Git providers on KIE Sandbox

### ISSUE THREAD

**Issue by thiagoelg** [apache/incubator-kie-issues#402]
Some enterprise instances of Git providers may have insecure/invalid TLS certificates and are being blocked by the browser when tried to be reached via KIE Sandbox.

Just like we have the option for OpenShift instances with self-signed certificates we need a way for users to allow insecure connections to Git providers (with a warning).

**Comment by thiagoelg** [comment]
Closing in favor of this one: https://github.com/kiegroup/kie-issues/issues/398
